src/quantum/index.jsx

```diff
@@ -99,12 +163,7 @@ export default class QuantumIndex extends React.Component {
               {
                 label: 'Firefox',
                 seriesConfig: {
-                  extraOptions: ['e10s', 'stylo'],
-                  framework: 1,
-                  platform,
-                  option: 'pgo',
-                  project: 'mozilla-central',
-                  suite: 'tp5o',
+                  and: [performanceFilter, { eq: { suite: 'tp5o' } }],
```

Comment:
This goes from human intelligible to not. No need to defend your position. If you insist on this approach please find another reviewer.

Reply:
What's happening here is I am defining a common factor `performanceFilter`.

If we were using the original `seriesConfig` structure, I would do something like:

```js
const performanceFilter ={
    extraOptions: ['e10s', 'stylo'],
    framework: 1,
    platform,
    option: 'pgo',
    project: 'mozilla-central'
}
```
 and then using that common factor in each `seriesConfig`:

```js
seriesConfig: {
    ...performanceFilter,
    suite: 'tp5o'
}
```

the same is happening here, only the logic is explicit:

```
seriesConfig: {
    and: [performanceFilter, { eq: { suite: 'tp5o' } }]
}
```

I added the common factor because the linter expands JSON over many lines, and I wanted to avoid the redundancy. The future hope is the whole array of performance tests will be generated from a configuration; multiplying the platforms by the suites, much like the quantum pages are generated now. I did not go that far here so it is easier to review the steps to that new code.